modules/parent-join/src/internalClusterTest/java/org/elasticsearch/join/query/ChildQuerySearchIT.java

```diff
@@ -1405,23 +1389,23 @@ public void testParentChildQueriesViaScrollApi() throws Exception {
             boolQuery().must(matchAllQuery()).filter(hasParentQuery("parent", matchAllQuery(), false)) };
 
         for (QueryBuilder query : queries) {
-            SearchResponse scrollResponse = prepareSearch("test").setScroll(TimeValue.timeValueSeconds(30))
-                .setSize(1)
-                .addStoredField("_id")
-                .setQuery(query)
-                .execute()
-                .actionGet();
-
-            assertNoFailures(scrollResponse);
-            assertThat(scrollResponse.getHits().getTotalHits().value, equalTo(10L));
-            int scannedDocs = 0;
-            do {
-                assertThat(scrollResponse.getHits().getTotalHits().value, equalTo(10L));
-                scannedDocs += scrollResponse.getHits().getHits().length;
-                scrollResponse = client().prepareSearchScroll(scrollResponse.getScrollId()).setScroll(TimeValue.timeValueSeconds(30)).get();
-            } while (scrollResponse.getHits().getHits().length > 0);
-            clearScroll(scrollResponse.getScrollId());
-            assertThat(scannedDocs, equalTo(10));
+            assertScrollResponses(
```

Comment:
Do we need the timeout here? Can we maybe just remove the timeout from this test to not have to bother with timeouts? (those are a known tricky issue around ref counting actually ...)